FreeRTOS_IP.c

```diff
@@ -1377,6 +1386,20 @@ eFrameProcessingResult_t eReturned = eReleaseBuffer;
 			/* Interpret the received Ethernet packet. */
 			switch( pxEthernetHeader->usFrameType )
 			{
+			case ipPTP_FRAME_TYPE:
+				/* The Ethernet frame contains an PTP packet. */
+				if( pxNetworkBuffer->xDataLength >=
+                    (ipEXPECTED_EthernetHeader_t_SIZE + ipEXPECTED_PTPHeader__SIZE))
+				{
+					eReturned = ePTPProcessPacket( pxNetworkBuffer->pucEthernetBuffer );
+				}
+				else
+				{
+                    printf("PTP Packet too small, %d\n", pxNetworkBuffer->xDataLength);
```

Comment:
I think your tabs are goofed. Could you make sure you run lint on this? `make format`

Reply:
Yup. This repo of external code is not in our code style so using formatter results in  massive changes which we don't want to do.    Will follow up to adjust to this repos's format (eg. uses  actual tabstops vs spaces).